IdentityCore/src/requests/sdk/adal/MSIDLegacyTokenRequestProvider.m

```diff
@@ -103,6 +103,11 @@ - (MSIDSilentTokenRequest *)silentSSOExtensionTokenRequestWithParameters:(__unus
     return nil;
 }
 
+- (nullable MSIDSilentTokenRequest *)silentXpcTokenRequestWithParameters:(nonnull MSIDRequestParameters *)parameters forceRefresh:(BOOL)forceRefresh { 
+    // TODO: not implemented yet.
```

Comment:
Maybe "Not support" could be more accurate? :)

Reply:
Modified